### DISCUSSION TITLE
r/city-china-zhejiang-huzhou-deqing-moganshan

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#57] [Announcements]
# r/city-china-zhejiang-huzhou-deqing-moganshan

游记｜上海出发2小时，探寻莫干山的清凉秘境

https://blog.ops-coffee.cn/r/city-china-zhejiang-huzhou-deqing-moganshan

**Comment by hullhut** [comment]
这才是生活,
我家离莫干山也是180多公里,看得我很心动.,可惜我家小狗脚受伤了,也不让也去逛逛

**Reply by addlistener** [reply]
找朋友借一条小狗，哈哈

**Comment by leirenbaobao** [comment]
文笔越来越好了

**Comment by xinzhang-hello** [comment]
看上去也适合亲子游

**Comment by addlistener** [comment]
真的很惬意 🐶

**Comment by hsz0707** [comment]
你老婆对你貌似并不满意

**Comment by Mingyi** [comment]
导航显示距离莫干山180公里，驾车只要两小时。

真实开车肯定不止3个小时。至少3个半小时以上。
